webapp/src/components/modals/create_issue/create_issue.jsx

```diff
@@ -105,6 +105,37 @@ export default class CreateIssueModal extends PureComponent {
 
     handleIssueDescriptionChange = (issueDescription) => this.setState({issueDescription});
 
+    renderIssueAttributeSelectors = () => {
+        if (this.state.repo && this.state.repo.permissions && !this.state.repo.permissions.push) {
```

Comment:
Actually, according to https://docs.github.com/en/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization the required permission is `triage`.

Reply:
@hanzei The permissions retrieved from the API call [here](https://github.com/mattermost/mattermost-plugin-github/blob/2061de81547aba5706273b93f8cdf9a87b3ac1cf/server/plugin/api.go#L1011) does not include `triage`. The available permissions are:

* `admin`
* `push`
* `pull`

Also, please take a look at https://github.com/mattermost/mattermost-plugin-github/pull/343#discussion_r471754812

Reply:
You are correct, that API doesn't return `triage`. I'm wondering a bit why that is the case, but we can only work with what the API provides.